eth/watchers/senderwatcher.go

```diff
@@ -128,6 +134,11 @@ func (sw *SenderWatcher) Clear(addr ethcommon.Address) {
 	}
 }
 
+// SubscribeSenderInfoChange notifies subscribers when the sender info for a particular sender changes
+func (sw *SenderWatcher) SubscribeSenderInfoChange(sink chan<- ethcommon.Address) event.Subscription {
```

Comment:
The subscription returned by this method only sends updates on the sink channel when the sender's reserve is funded. Should the method name be `SubscribeReserveFunded()` instead?